src/test/intergration.test.js

```diff
@@ -0,0 +1,20 @@
+import { test, expect } from '@jest/globals';
+
+test('Check if a home cards are  displayed home page properly', () => {
+  const number = 24;
+
+  const isEven = number % 2 === 0;
+
+  if (isEven) {
+    expect(isEven).toBe(true);
+  } else {
+    expect(isEven).toBe(false);
+  }
+});
+
+test("Ball Don't Lie Team Stats", () => {
+  const teamStatsAreGreat = true;
+
```

Comment:
- [ ] I really appreciate the effort you've invested in testing your code. 🙌😊. However, It's crucial to ensure that our tests align with the specific requirements of our Redux implementation.

📦 Redux Actions: Test how Redux actions respond to different inputs and scenarios.
🔄 Reducers: Verify that reducers appropriately handle actions and update the state.
🧩 State Management: Ensure that the overall state management functions as expected, reflecting the changes caused by actions and reducers.

### The tests you've provided seem to be unrelated to our Redux code's functionality.